README.md

```diff
@@ -430,6 +430,7 @@ These variables must be set at the root or on a container element wrapping any b
 ```css
 html {
   --bc-color-brand: #196ce7; /* Only 6-digit hex and rgb formats are supported! */
+  --bc-color-brand-text: #ffffff; /* override text color for buttons coloured in brand colour. Default is otherwise chosen between white and black based on luminance of the colour defined in --bc-color-brand*/
```

Comment:
I am also wondering about the naming, since it only applies to the button. Do you see this being applied elsewhere?

Reply:
maybe naming could be more specific for only the button case, --bc-color-brand-primary-button-text? I couldnt see any other places, at least for our current use case, where we would need to change text colour specifically

Reply:
Maybe `--bc-color-brand-button-text` would be good for now. In this case the brand color is the primary color

Reply:
And `--bc-color-brand-button-text-dark` for the dark version